Wikipedia/Code/WMFArticleContainerViewController+TOC.swift

```diff
@@ -19,19 +19,29 @@ extension WMFArticleContainerViewController : WMFTableOfContentsViewControllerDe
 
     public func tableOfContentsController(controller: WMFTableOfContentsViewController,
                                           didSelectItem item: TableOfContentsItem) {
-
+        var dismissVCCompletionHandler: (() -> Void)?
         if let section = item as? MWKSection {
             // HAX: webview has issues scrolling when browser view is out of bounds, disable animation if needed
             self.webViewController.scrollToSection(section, animated: self.webViewController.isWebContentVisible)
+            dismissVCCompletionHandler = {
+                // HAX: This is terrible, but iOS events not under our control would steal our focus if we didn't wait long enough here and due to problems in UIWebView, we cannot work around it either.
+                dispatchOnMainQueueAfterDelayInSeconds(1) {
+                    self.webViewController.accessibilityCursorToSection(section)
```

Comment:
@bgerstle I was thinking around a third of a second for the same reason. If @hhanke agrees I think it's good.